infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java

```diff
@@ -71,43 +77,43 @@
 
     private final static int GRAPH_ENDPOINT_INDEX = 7;
 
-    private final static int RM_HOSTNAME_INDEX = 8;
+    private final static int RM_HTTP_URL_INDEX = 8;
 
     private final static int CONNECTOR_IAAS_URL_INDEX = 9;
 
     private final static int IMAGE_INDEX = 10;
 
-    private final static int VM_SIZE_TYPE_INDEX = 11;
+    private final static int IMAGE_OS_TYPE_INDEX = 11;
+
+    private final static int VM_SIZE_TYPE_INDEX = 12;
 
-    private final static int VM_USERNAME_INDEX = 12;
+    private final static int VM_USERNAME_INDEX = 13;
 
-    private final static int VM_PASSWORD_INDEX = 13;
+    private final static int VM_PASSWORD_INDEX = 14;
 
-    private final static int VM_PUBLIC_KEY_INDEX = 14;
+    private final static int VM_PUBLIC_KEY_INDEX = 15;
 
-    private final static int RESOURCE_GROUP_INDEX = 15;
+    private final static int RESOURCE_GROUP_INDEX = 16;
 
-    private final static int REGION_INDEX = 16;
+    private final static int REGION_INDEX = 17;
 
-    private final static int NUMBER_OF_INSTANCES_INDEX = 17;
+    private final static int NUMBER_OF_INSTANCES_INDEX = 18;
 
-    private final static int NUMBER_OF_NODES_PER_INSTANCE_INDEX = 18;
+    private final static int NUMBER_OF_NODES_PER_INSTANCE_INDEX = 19;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)